crates/rolldown/tests/esbuild/packagejson/test_package_json_syntax_error_trailing_comma/artifacts.snap

```diff
@@ -0,0 +1,37 @@
+---
+source: crates/rolldown/tests/common/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/packagejson/test_package_json_syntax_error_trailing_comma
+---
+# Errors
+
+## ANYHOW_Error
+
+```text
+[ANYHOW_Error] Error: Anyhow error: Unexpectedly failed to resolve dependencies of /src/entry.js. Got errors [
```

Comment:
Passed tests should not return any `anyhow::Error`.

> Panics/`anyhow::Error` are usually program errors and can't not be handled, so it's always unexpected. If some errors are wrongly considered as programs errors, we should fix the behavior rather than thinking programs errors are expected. https://github.com/rolldown/rolldown/pull/1224#issuecomment-2126882955